publify_core/app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -22,10 +22,10 @@
   padding: 45px;
   border: 1px solid #ccc;
 }
-.sidebar_item {
+.sidebar-item {
   position: relative;
   h3 {
-    margin-top: 0;
+    margin-top: 100px;
```

Comment:
Why is this necessary?

Reply:
Oh This is not nessesary. I texted it before we figured out. I forgot to deleted it